### PR TITLE
Feat[15632861]: change tooltip translations

### PR DESCRIPTION
**Pull request by KornelKwiatkowski**
## Description

- Task: https://tw.auroracreation.com/app/tasks/feat/

**Changes** 🏗

Changed translations for tooltips

(Resolves | Contributes to) #31415

### CODE REVIEW

**Review by TomaszChmielPagepro — APPROVED** [review]

File: src/i18n/en.json [hunk 1]
```diff
@@ -964,49 +964,87 @@
...
-  "actionSidebar.tooltip.editColony.colonyName": "The display name of the Colony.",
-  "actionSidebar.tooltip.editColony.colonyLogo": "The logo of the Colony.",
-  "actionSidebar.tooltip.editColony.colonyDescription": "The short description of the Colony what it is about and it's purpose.",
+  "actionSidebar.tooltip.actionType": "Identifies the type of action.",
+  "actionSidebar.tooltip.decisionMethod": "Mechanism behind the decision-making process.",
+  "actionSidebar.tooltip.createdIn": "Team responsible for decision-making and action execution.",
+  "actionSidebar.tooltip.description": "Rationale and context behind the action.",
+  "actionSidebar.tooltip.singlePayment.from": "Team source of the funds.",
+  "actionSidebar.tooltip.singlePayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.singlePayment.amount": "The amount of funds and token type for the payment.",
```
Comment (arrenv) [inline src/i18n/en.json:973]: As of the spec here https://www.notion.so/colony/Actions-Motions-775cb0e78eb44783a6f28702b7a9dc33?pvs=4#64368b1c05044078a4374f755be26754 and here https://tw.auroracreation.com/app/tasks/15567878?c=5830182

This should be: "Quantity and type of funds of the payment."

File: src/i18n/en.json [hunk 2]
```diff
@@ -964,49 +964,87 @@
...
+  "actionSidebar.tooltip.batchPayment.token": "Token type used for the payment.",
+  "actionSidebar.tooltip.splitPayment.fundFrom": "Team source of the payment.",
+  "actionSidebar.tooltip.splitPayment.description": "The way in which the payment should be divided.",
+  "actionSidebar.tooltip.splitPayment.amount": "Quantity and type of funds of the payment.",
+  "actionSidebar.tooltip.splitPayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.splitPayment.percent": "Proportional division of funds for recipients.",
+  "actionSidebar.tooltip.stagedPayment.fundFrom": "Team source of the payment.",
+  "actionSidebar.tooltip.stagedPayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.stagedPayment.milestone": "Specific phase or stage associated with the payment.",
+  "actionSidebar.tooltip.stagedPayment.amount": "The amount of funds and token type for the milestone.",
```
Comment (arrenv) [inline src/i18n/en.json:991]: Can this instead be: "Quantity and type of funds for the milestone."

File: src/i18n/en.json [hunk 3]
```diff
@@ -964,49 +964,87 @@
...
+  "actionSidebar.tooltip.editTeam.team.colour": "The color used to represent the team.",
+  "actionSidebar.tooltip.manageReputation.team": "The team reputation is being managed in.",
+  "actionSidebar.tooltip.manageReputation.modification": "The type of modification being made to the members reputation.",
+  "actionSidebar.tooltip.manageReputation.member": "Member or address for the reputation change.",
+  "actionSidebar.tooltip.manageReputation.currentAmount": "Existing amount of reputation.",
+  "actionSidebar.tooltip.manageReputation.newAmount": "New amount of reputation after update.",
+  "actionSidebar.tooltip.manageVerifiedMembers.member": "Member or address being verified.",
+  "actionSidebar.tooltip.managePermissions.member": "Member or address for the permission change.",
+  "actionSidebar.tooltip.managePermissions.team": "The team permissions is being managed in.",
+  "actionSidebar.tooltip.managePermissions.permissions": "The permission for the selected member. View permissions list. INSTRUCTIONS: View permissions list - should open the modal",
```
Comment (arrenv) [inline src/i18n/en.json:1022]: The "INSTRUCTIONS" are what need to be done, not to be included. The text should be:
"The permission for the selected member. View permissions list."

With the text "View permissions list", opening the full permissions modal.

**Review by arrenv — CHANGES_REQUESTED** [review]
Thank you Kornel, please see comments for required changes.

File: src/i18n/en.json (hunk 1 above)
Comment (KornelKwiatkowski) [reply src/i18n/en.json:973]: done

File: src/i18n/en.json (hunk 2 above)
Comment (KornelKwiatkowski) [reply src/i18n/en.json:991]: done

File: src/i18n/en.json (hunk 3 above)
Comment (KornelKwiatkowski) [reply src/i18n/en.json:1022]: done

Comment (arrenv) [reply src/i18n/en.json]: If the "View permissions list." part of the text is not able to be clickable in the popover, can we just remove it please.

Comment (KornelKwiatkowski) [reply src/i18n/en.json]: Ok, removed :)

**Review by arrenv — APPROVED** [review]
Thank you for correcting, looks good.

**Pull request merged**